docs/docs-beta/docs/guides/external-systems/pipes.md

```diff
@@ -1,5 +1,33 @@
 ---
-title: "Excuting code outside of Dagster with Pipes"
+title: "Executing code outside of Dagster with Pipes"
 sidebar_position: 40
 sidebar_label: "External execution with Pipes"
----
\ No newline at end of file
+---
+
+# Executing code outside of Dagster with Pipes
+
+Dagster Pipes provides a powerful mechanism for invoking code outside of Dagster, while providing all the benefits of scheduling, reporting, and observability of native Dagster pipelines.
+
+In this guide, we'll walk you through how to invoke non-Dagster code through Pipes.
+
+<details>
+<summary>Prerequisites</summary>
+
+- Familiarity with [Assets](/concepts/assets)
+</details>
+
+## Setting up an asset that invokes your external code
+
+To set up invoking code outside of Dagster, you first need to set up an asset wrapper for that external code invocation.  We can invoke the code using a Dagster Pipes client resource.
```

Comment:
we should probably call out that no external code changes required in this step. that's a general mis-impression of pipes [1]